konrad/atmosphere.py

```diff
@@ -266,6 +266,7 @@ def copy(self):
         """
         datadict = dict()
         datadict['plev'] = copy(self['plev'])  # Copy pressure grid.
```

Comment:
I think one does not even need ``plev`` as it is calculated from ``phlev`` when initialising the ``Atmosphere`` object. If you want, you can simply replace ``plev`` with ``phlev`` in this line 😉